main.go

```diff
@@ -16,6 +18,9 @@ import (
 func main() {
 	log.Info("GlusterD starting")
 
+	utils.InitDir(config.LocalStateDir)
+	context.MyUUID = context.InitMyUUID()
```

Comment:
Yes, why can't we? This change was done to ensure that we initialize MyUUID such that `peer.AddSelfDetails()` can refer to it and store its own information into the store.